paddle/fluid/framework/ir/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ cc_library(fuse_pass_base SRCS fuse_pass_base.cc DEPS pass)
 pass_library(graph_to_program_pass base)
 pass_library(graph_viz_pass base)
 pass_library(lock_free_optimize_pass base)
+pass_library(int8_placement_pass base)
```

Comment:
to mkldnn_int8_placement_pass ?

Reply:
I personally don't suggest to name it as "mkldnn_int8" considering we may use various mechanism (int8 mkl, int8 mkldnn, etc.) to implement int8 kernel.

Reply:
@wojtuss Do you prefer `mkldnn_int8_placement_pass` as well?

Reply:
@wojtuss @Superjomn @jianhang-liu How about `cpu_int8_placement_pass`?